<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DOLTab</class>
 <widget class="QWidget" name="DOLTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="import_dol">
       <property name="text">
        <string>Import DOL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_dol">
       <property name="text">
        <string>Export DOL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>RAM Address: 0x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dol_address"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="convert_from_dol_offset">
         <property name="text">
          <string>&lt;- Convert offset to address &lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="convert_from_dol_address">
         <property name="text">
          <string>-&gt; Convert address to offset -&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Offset in DOL: 0x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dol_offset"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="dol_sections_tree"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
